e2e/src/test/groovy/gov/hhs/cdc/trustedintermediary/e2e/OrderTest.groovy

```diff
@@ -52,6 +52,7 @@ class OrderTest extends Specification {
         parsedSentPayload.entry[0].resource.remove("eventCoding")
         parsedLabOrderJsonFile.entry[0].resource.remove("eventCoding")
         parsedSentPayload.entry[24].resource.remove("contact")
+        parsedSentPayload.entry[0].resource.meta.tag.remove(1)
```

Comment:
#### Note
I need to include the meta tag in the comment on top of these lines of code.

Reply:
done